### PR TITLE
Support avalonia master/nightly

### PR DESCRIPTION
**Pull request by AsafMahlev**
Fixes https://github.com/AvaloniaUI/Avalonia.HtmlRenderer/issues/16

You probably shouldn't merge this to master, but have a dev branch like avalonia editor.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia.HtmlRenderer?pullRequest=17) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia.HtmlRenderer?pullRequest=17) before we can accept your contribution.<br/><br/>:x: AsafMahlev [sign now](https://cla.dotnetfoundation.org/AvaloniaUI/Avalonia.HtmlRenderer?pullRequest=17)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/AvaloniaUI/Avalonia.HtmlRenderer?pullRequest=17) it.</sub>

File: Adapters/ControlAdapter.cs [hunk 1]
```diff
@@ -42,6 +48,38 @@ public ControlAdapter(Control control)
...
+                        break;
+                    case MouseButton.Right:
+                        _rightMouseButton = false;
+                        break;
+                }
+            };
+            
+            _control.ContextRequested += (sender, args) =>
+            {
+                _rightMouseButton = true;
```
Comment (maxkatz6) [inline Adapters/ControlAdapter.cs:81]: ContextRequested will also be called when users opens a context with a platform depended hotkey. Is it expected?

File: HtmlControl.cs [hunk 2]
```diff
@@ -411,7 +416,14 @@ protected override void OnPointerPressed(PointerPressedEventArgs e)
         {
             base.OnPointerPressed(e);
             LeftMouseButton = true;
-            _htmlContainer?.HandleLeftMouseDown(this, e);
+            if (e.ClickCount == 1)
+            {
+                _htmlContainer?.HandleLeftMouseDown(this, e);
+            }
+            else if (e.ClickCount == 2)
```
Comment (maxkatz6) [inline HtmlControl.cs:423]: I think it's better to use DoubleTapped event instead, so it can be handled with touch better

Comment (AsafMahlev) [reply HtmlControl.cs:423]: HandleMouseDoubleClick expects PointerEventArgs, which we don't get on DoubleTapped, so I'm not sure how to pass it this data.

File: Adapters/ControlAdapter.cs (hunk 1 above)
Comment (AsafMahlev) [reply Adapters/ControlAdapter.cs:81]: Seems reasonable, as it is equivalent, isn't it?

Comment (maxkatz6) [reply Adapters/ControlAdapter.cs:81]: Yes, as for me that's fine. Just wanted to confirm.

File: HtmlControl.cs (hunk 2 above)
Comment (maxkatz6) [reply HtmlControl.cs:423]: It needs only current position
https://github.com/AvaloniaUI/Avalonia.HtmlRenderer/blob/b3997448d3f57823eff6f2190a80643bdfc5102d/HtmlContainer.cs#L383
Which you can get with same method from TappedEventArgs
https://github.com/AvaloniaUI/Avalonia/blob/fb0da85ad3a998bdf711b42e7b619550e6ff9de3/src/Avalonia.Base/Input/TappedEventArgs.cs#L20

**Comment by maxkatz6** [comment]
Thanks! Completed in #20.

**Pull request closed without merging**